kafka_consumer/check.py

```diff
@@ -177,6 +186,11 @@ def check(self, instance):
 
         # Report the consumer group offsets and consumer lag
         for (consumer_group, topic, partition), consumer_offset in consumer_offsets.iteritems():
+            if (topic, partition) not in highwater_offsets:
+                self.log.debug("[%s] topic: %s partition: %s was not available in the broker "
```

Comment:
this seems to be the exact same condition as 5 lines below. Should we remove the other block and log this message at the `warn` level? (i.e. same level as the other block, unless such a case can be expected)

Reply:
missed that - good eye as always 👍